core/services/vrf/delegate.go

```diff
@@ -73,6 +74,10 @@ func (d *Delegate) OnDeleteJob(job.Job, pg.Queryer) error { return nil }
 
 // ServicesForSpec satisfies the job.Delegate interface.
 func (d *Delegate) ServicesForSpec(jb job.Job) ([]job.ServiceCtx, error) {
+
+	// TODO
+	ctx := context.Background()
```

Comment:
Can we modify the method signature to pass this from up the stack?

Reply:
I did in the latest revision - it wasn't as bad as I originally thought to update the job.Delegate interface. I stopped here to somewhat draw a line in the sand (for the scope of the PR) because implementations of ServicesForSpec that needed context would go and initialize a background context anyways.